src/dockerflow/fastapi/views.py

```diff
@@ -11,12 +11,12 @@ def lbheartbeat():
     return {"status": "ok"}
 
 
-def heartbeat(request: Request, response: Response):
+async def heartbeat(request: Request, response: Response):
```

Comment:
I'm wondering if it's important to give users the choice to run these pre-baked views (and in this case, the underlying checks) synchronously or asynchronously, or if we should go all in on async at some point and make all of these FastAPI views async 🤔

Reply:
I can't think of any real harm with allowing both sync/async checks to be ran. In general, I think async always makes more sense in an API context. But if some checks are simple things like "Do I have proper context/config?", then yeah that won't really be async.